contracts/cw-storage/src/error.rs

```diff
@@ -17,4 +17,19 @@ pub enum ContractError {
 pub enum BucketError {
     #[error("Name of bucket could not be empty")]
     EmptyName,
+
+    #[error("Maximum total size exceeded")]
+    MaxTotalSizeLimitExceeded,
```

Comment:
I would tend to bring more context to the errors. For instance:

```suggestion
    #[error("Maximum objects number exceeded: {0} / {1}")]
    MaxObjectsLimitExceeded(Uint128, Uint128),
```